src/styles/_catalog.scss

```diff
@@ -1,22 +1,44 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: $cardWidth;
+  gap: $cardGap;
 
   max-width: 944px;
   margin: 0 auto;
 
   &__card {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    width: 200px;
-    height: 300px;
+    width: $cardWidth;
+    height: $cardHeight;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
+    // margin-bottom: $cardGap;
+    
+    line-height: $cardHeight;
+    text-align: center;
   }
 
   &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+    margin-right: $cardGap;
+  }
+}
+
+@media (min-width: 488px) {
+  .catalog {
+    display: grid;
```

Comment:
try play a bit with and comment this row first, look on browser or better use dev tool for sure. after you can delete it at all.